src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedAutoPropAccessorSymbol.cs

```diff
@@ -0,0 +1,205 @@
+                    Debug.Assert(MethodKind == MethodKind.PropertySet);
+                    return _voidType.Value;
+                }
+
+                return _property.TypeWithAnnotations;
+            }
+        }
+
+        public override MethodKind MethodKind => _accessorKind == AccessorKind.Get
+            ? MethodKind.PropertyGet
+            : MethodKind.PropertySet;
+
+        internal override bool IsInitOnly => _accessorKind == AccessorKind.Init;
+
+        public override int Arity => 0;
+
+        public override bool IsExtensionMethod => false;
+
+        public override bool HidesBaseMethodsByName => false;
+
+        public override bool IsVararg => false;
+
+        public override bool ReturnsVoid => _voidType is object;
+
+        public override bool IsAsync => false;
+
+        public override RefKind RefKind => RefKind.None;
+
+        public override FlowAnalysisAnnotations ReturnTypeFlowAnalysisAnnotations => FlowAnalysisAnnotations.None;
+
+        public override ImmutableArray<TypeWithAnnotations> TypeArgumentsWithAnnotations => ImmutableArray<TypeWithAnnotations>.Empty;
+
+        public override ImmutableArray<TypeParameterSymbol> TypeParameters => ImmutableArray<TypeParameterSymbol>.Empty;
+
+        public override ImmutableArray<ParameterSymbol> Parameters
+        {
+            get
+            {
+                if (_accessorKind == AccessorKind.Get)
+                {
+                    return ImmutableArray<ParameterSymbol>.Empty;
+                }
+                return ImmutableArray.Create(SynthesizedParameterSymbol.Create(
+                    this,
+                    _property.TypeWithAnnotations,
+                    ordinal: 0,
+                    RefKind.None,
+                    name: ParameterSymbol.ValueParameterName));
+            }
+        }
+
+        public override ImmutableArray<MethodSymbol> ExplicitInterfaceImplementations => ImmutableArray<MethodSymbol>.Empty;
+
+        public override ImmutableArray<CustomModifier> RefCustomModifiers => _property.RefCustomModifiers;
+
+        public override Symbol AssociatedSymbol => _property;
+
+        public override Symbol ContainingSymbol => _property.ContainingSymbol;
+
+        public override ImmutableArray<Location> Locations => _property.Locations;
+
+        public override Accessibility DeclaredAccessibility => _property.DeclaredAccessibility;
+
+        public override bool IsStatic => _property.IsStatic;
+
+        public override bool IsVirtual => _property.IsVirtual;
+
+        public override bool IsOverride => _property.IsOverride;
+
+        public override bool IsAbstract => _property.IsAbstract;
+
+        public override bool IsSealed => _property.IsSealed;
+
+        public override bool IsExtern => _property.IsExtern;
+
+        public override ImmutableHashSet<string> ReturnNotNullIfParameterNotNull => ImmutableHashSet<string>.Empty;
+
+        internal override bool HasSpecialName => _property.HasSpecialName;
```

Comment:
>_property.HasSpecialName [](start = 49, length = 24)

I don't think there is a relationship like that. In fact, it looks like property accessors are always emitted with specialname flag.